api.md

```diff
@@ -1229,6 +1230,59 @@ For a table with more than one space dimension:
 SELECT set_number_partitions('conditions', 2, 'device_id');
 ```
 
+
+---
+
+## set_replication_factor() [](set_replication_factor)
+Sets the replication factor of a distributed hypertable to the given value. 
+Changing the replication factor does not affect the number of replicas for existing chunks.
+Chunks created after changing the replication factor will be replicated 
+in accordance with new value of the replication factor. If the replication factor cannot be
+satisfied, since the amount of attached data nodes is less than new replication factor, 
+the command aborts with an error.
+
+If existing chunks have less replicas than new value of the replication factor,
+the function will print a warning.
+
+#### Required Arguments [](set_replication_factor-required-arguments)
+
+|Name|Description|
+|---|---|
+| `hypertable` | (REGCLASS) Identifier of the distributed hypertable to update the replication factor for.|
+| `replication_factor` | (INTEGER) The new value of the replication factor. Must be greater than 0, 
```

Comment:
Normally adding a newline here would be good, to keep the file properly readable.  Here though, it disrupts the table formatting:
<img width="827" alt="Screen Shot 2020-01-23 at 9 49 54 AM" src="https://user-images.githubusercontent.com/12288273/72994776-bbfd1b00-3dc5-11ea-801c-3cb9bf595a28.png">

Reply:
Will remember for future. Fixed.